test/commute.js

```diff
@@ -17,7 +16,26 @@ describe('commute', function() {
   });
 
   it('works on Algebraic Data Types such as "Maybe"', function() {
```

Comment:
Sure. Then I would suggest removing the array examples. They are just confusing to me and imply that this is the main usage.